### DISCUSSION TITLE
Google map or GTA 5 type map in Carla

### DISCUSSION THREAD

**Discussion by dragnil1** [carla-simulator/carla#6253] [Q&A]
Hello, I am new to Carla. Is there a way I can get the image of map indicating where I am and the navigation path (like GTA 5 or Google map directions) in Carla.

**Comment by tmnku** [comment]
Maybe have a look at this [example](https://github.com/carla-simulator/carla/blob/master/PythonAPI/examples/no_rendering_mode.py) on how to draw a map of the city. Then you can set up a routing graph using the information from `waypoint_list = map.generate_waypoints(waypoint_spacing)` and `waypoint_tuple_list = map.get_topology()`. The shortest path to your destination can be found using some graph search algorithm of your choice.
